modes/ripgrep/evil-collection-ripgrep.el

```diff
@@ -0,0 +1,50 @@
+;;; evil-collection-ripgrep.el --- Bindings for `ripgrep' -*- lexical-binding: t -*-
+
+;; Copyright (C) 2018 James Nguyen
+
+;; Author: Sid Kasivajhula <sid@countvajhula.com>
+;; Maintainer: James Nguyen <james@jojojames.com>
+;; Pierre Neidhardt <mail@ambrevar.xyz>
+;; URL: https://github.com/emacs-evil/evil-collection
+;; Version: 0.0.1
+;; Package-Requires: ((emacs "25.1"))
+;; Keywords: evil, emacs, tools
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; Bindings for `ripgrep'.
+
+;;; Code:
+(require 'evil-collection)
+(require 'ripgrep nil t)
+
+(defconst evil-collection-ripgrep-maps '(ripgrep-search-mode-map))
+
+;;;###autoload
+(defun evil-collection-ripgrep-setup ()
+  "Set up `evil' bindings for `ripgrep'."
+  (evil-collection-define-key 'normal 'ripgrep-search-mode-map
+    "n" 'evil-search-next
+    "\C-k" 'previous-error-no-select
+    "\C-j" 'next-error-no-select
+    "gk" 'previous-error-no-select
+    "gj" 'next-error-no-select
+    "{" 'compilation-previous-file
```

Comment:
Not super sure about this off the top of my head. I'd recommend looking at other files for inspiration. (If other files use these bindings, feel free to resolve.)

Reply:
re: { and }

Reply:
I lifted that one [from rg](https://github.com/emacs-evil/evil-collection/blob/5151be9b633a8b530d6050e55178a7b5056c10cc/modes/rg/evil-collection-rg.el#L62-L63).